### ISSUE TITLE
Add external posts

### ISSUE THREAD

**Issue by bebatut** [open-life-science/open-life-science.github.io#778]
- program launch: https://www.software.ac.uk/blog/2019-11-22-supercharge-your-open-project-leadership-training
- on mentoring: https://www.software.ac.uk/blog/2020-10-09-key-skills-know-mentoring
- eosc: https://www.eosc-life.eu/news/4-projects-awarded-funding-within-first-training-open-call/ and https://www.eosc-life.eu/news/training-open-science-for-eosc-life-ris/
- galaxy: https://galaxyproject.eu/posts/2020/12/22/ols/

**Comment by ikayz** [comment]
@bebatut where on the site should these posts be? And is this still something that needs to be done?

**Comment by yochannah** [comment]
Hmmm, maybe this should be combined with #777? "Mentions of OLS and external posts"? "Literature?" "OLS on other sites?"

@NPDebs @bebatut naming is hard - any suggestions?

**Comment by ikayz** [comment]
@yochannah has the name been decided for the page? How about "Mentions"?

**Comment by NPDebs** [comment]
@ikayz - Nice to GitHub-meet you! Thanks for offering to pick this up. 🙏 

@yochannah - Naming **is** hard. I do like the sound of **Mentions**. Other suggestions - 
- Features
- External recognition
- Shoutouts

We can always start with one name and let it evolve over time.

**Comment by ikayz** [comment]
@NPDebs nice to GitHub-meet you too! Those are great suggestions, I can start with whichever name everyone agrees on.

**Comment by NPDebs** [comment]
> @NPDebs nice to GitHub-meet you too! Those are great suggestions, I can start with whichever name everyone agrees on.

@ikayz — To avoid consensus becoming a blocker, it might be better if you start with the name you suggested (i.e mentions). We can always adjust things as we go.

What say you?

**Comment by ikayz** [comment]
@NPDebs that sounds good, I'll get started right away

**Comment by NPDebs** [comment]
> @NPDebs that sounds good, I'll get started right away

Thank you!! ✨

**Comment by ikayz** [comment]
@NPDebs would you assign this task to me?

**Comment by NPDebs** [comment]
It's all yours, @ikayz! Thanks again for taking this up.

**Comment by ikayz** [comment]
@NPDebs thank you.

**Comment by SaraVilla** [comment]
Hi @ikayz ! just checking if you're still thinking about this issue, since I'm tidying up a bi tour issues board... :) 
Also, please, do let me know if I can help somehow! 
thanks!

**Comment by SaraVilla** [comment]
close and refer back when needed

**Comment by ikayz** [comment]
@SaraVilla yes I am, the challenge I had was with making changes to repo — as they were not reflecting.
